src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
```

Comment:
It is ok to use empty lines between multiline sibling blocks of HTML, but better to not have them between parent and child elements